enginetest/join_op_tests.go

```diff
@@ -203,6 +203,39 @@ var joinOpTests = []struct {
 				Query:    "select * from xy where x = 1 and x in (select y from xy union select 1)",
 				Expected: []sql.Row{{1, 1}},
 			},
+			{
+				Query: "select * from xy where x = 1 intersect select * from uv;",
```

Comment:
could you do one with each with the format i have on line 203? in that gets converted into semi join?